config/api.yml

```diff
@@ -929,17 +929,27 @@
     - :collection
     :verbs: *gp
     :klass: MiqRequest
+    :subcollections:
+    - :request_tasks
+    - :tasks
     :collection_actions:
       :get:
       - :name: read
         :identifier: miq_request_show_list
       :post:
       - :name: query
         :identifier: miq_request_show_list
+      - :name: create
+        :identifier: vm_miq_request_new
```

Comment:
done,  role identifier per request type now provided by model and used for authorization.